articles/logging.md

```diff
@@ -0,0 +1,75 @@
+## Desired User Experience
+This section describes the desired user interface of the ROS2 logging system.
+To define the interface of the ROS2 logging system, it is helpful to consider all of the various tasks users will want to perform.
+From this list of tasks, we can describe the steps users will have to take in order to perform each one.
+Based on our experiences with ROS1, other logging systems, and community feedback, the logging system in ROS2 will support the following interactions:
+- creating loggers
+- setting log levels
+- outputting logs
+- defining output destinations
+- specifying a backend
+- implementing a custom backend
+- tailoring performance impact
+### Interaction Descriptions
+This section describes what each of the interactions in the list entails, and the steps users will have to take to complete them.
+#### Creating Loggers
+This interaction contitutes creating the objects that users will use to output information from their code.
+#### Setting Log Levels
+This is how users set the severity levels for the loggers in their code.
+There are a number of ways users might want to be able to change these levels, so this interaction is further broken down to describe each of those.
+##### Programatically
+##### Command Line
+##### Via ROS2 Service Call
+##### Environment Variable?
```

Comment:
This seems more like a solution/implementation detail, rather than a thing we want users to be able to do.  As such, I'd leave it out of this section.